tests/system_tests/test_registries/test_registry.py

```diff
@@ -0,0 +1,234 @@
+import pytest
+from wandb import Api
+from wandb.sdk.artifacts._validators import REGISTRY_PREFIX
+
+
+def test_registry_create_edit(user_in_org):
+    """Tests the basic CRUD operations for a registry."""
+    organization = user_in_org.organization_name
+    api = Api()
+    registry_name = "test"
+    initial_description = "Initial registry description."
+    updated_description = "Updated registry description."
+    artifact_type_1 = "model-1"
+
+    # TODO: Setting visibility to restricted is giving permission errors.
+    # Need to dig into backend code to figure out why as local testing works fine.
```

Comment:
I think this is some deeper auth issue with the backend fixture. Going to table this for another PR to fix/investigate. Like mentioned testing this locally worked just fine